osu.Game/Beatmaps/BeatmapShortInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Game.Beatmaps
+{
+    public class BeatmapShortInfo : IEquatable<BeatmapShortInfo>
```

Comment:
Cannot exist with this name and in this namespace. Should be a structure local to the mod overlay.

Probably something like `osu.Game.Overlays.Mods.ModEffectPreviewInfo` or something.

Also, for bonus points, if you make this a `struct` or a `record` (whichever is more convenient), you can skip the `Equals()` implementation.

Reply:
deleted this completely